pkgs/applications/version-management/blackbox/default.nix

```diff
@@ -13,13 +13,13 @@
 
 stdenv.mkDerivation rec {
   pname = "blackbox";
-  version = "2.0.0";
+  version = "1.20220610";
```

Comment:
```suggestion
  version = "unstable-2022-06-10";
```

Reply:
Is there a reason to choose `unstable-`? Their latest tag is [`v1.20220610`](https://github.com/StackExchange/blackbox/tags), which seems to be used by [other repositories](https://repology.org/project/blackbox-vcs/versions)

Reply:
Sorry, I misinterpreted your comment. I thought they removed  **all** tags.